test/test-network/systemd-networkd-tests.py

```diff
@@ -337,6 +338,26 @@ def remove_networkd_conf_dropin(*dropins):
 def clear_networkd_conf_dropins():
     rm_rf(networkd_conf_dropin_dir)
 
+def setup_systemd_udev_rules():
+    if not systemd_udev_rules_dir:
+        return
+
+    # Copy non-templated rule files from the source tree as well
+    src_udev_rules = os.path.join(os.path.dirname(os.path.abspath(__file__)), "../../rules.d/")
```

Comment:
It may be overkill, but I'd like to add --git-tree or so to specify the directory, and fallback to this logic if it doesn't specified.